addons/l10n_fr/i18n/l10n_fr.pot

```diff
@@ -3258,6 +3288,191 @@ msgstr ""
 msgid "Sécurité Sociale"
 msgstr ""
 
+#. module: l10n_fr
+#: model:account.tax.report.line,name:l10n_fr.tax_report_T1_base
+msgid ""
+"T1 - Opérations réalisées dans les DOM et imposables au taux de 1,75% (base)"
+msgstr ""
+
+#. module: l10n_fr
+#: model:account.tax.report.line,name:l10n_fr.tax_report_T1_taxe
+msgid ""
+"T1 - Opérations réalisées dans les DOM et imposables au taux de 1,75% (taxe)"
+msgstr ""
+
+#. module: l10n_fr
+#: model:account.tax.report.line,tag_name:l10n_fr.tax_report_T1_base
+msgid "T1_base"
+msgstr ""
+
+#. module: l10n_fr
+#: model:account.tax.report.line,tag_name:l10n_fr.tax_report_T1_taxe
+msgid "T1_taxe"
+msgstr ""
+
+#. module: l10n_fr
+#: model:account.tax.report.line,name:l10n_fr.tax_report_T2_base
+msgid ""
+"T2 - Opérations réalisées dans les DOM et imposables au taux de 1,05% (base)"
+msgstr ""
```

Comment:
It is supposed to be sorted alphabetically

Reply:
well `"T1_taxe" < "T2 - Opérations"` is `True`, so I would say it's correct like this ?